policy/release/test.rego

```diff
@@ -222,6 +223,40 @@ deny contains result if {
 	result := lib.result_helper(rego.metadata.chain(), [error])
 }
 
+# METADATA
+# title: Verify image disgest is present in IMAGES_PROCESSED result
+# description: >-
+#   Validates that each task with the IMAGES_PROCESSED result contains the
+#   digest of the image beind processed.
+# custom:
+#   short_name: image_digest_processed
+#   failure_msg: Test '%s' is missing '%s' from IMAGES_PROCESSED result.
+#   solution: >-
+#     There is a task with result `IMAGES_PROCESSED` which did not scan
+#     a digest. Check that all tasks with `IMAGES_PROCESSED` result has
+#     processed every digest of the input image(s).
+#   collections:
+#   - redhat
+#
+deny contains result if {
+	input_image := image.parse(input.image.ref)
+	input_image_digest := input_image.digest
+	unprocessed_digest_task := [unprocessed_task |
+		some x_result in lib.images_processed_results_from_tests
+		result_map := lib.unmarshal(x_result.value)
```

Comment:
`lib.results_named` should already unmarshal (if valid JSON), so `.value` should be an object here.